app/components/Breadcrumb.js

```diff
@@ -21,6 +22,8 @@ import Flex from "components/Flex";
 import BreadcrumbMenu from "./BreadcrumbMenu";
 import { collectionUrl } from "utils/routeHelpers";
 
+const t = (k) => i18n.t(k);
```

Comment:
Why not use the hook in here?

Reply:
I think the hook can only be used on functional components - Maybe i am wrong?

Can you help me with this?

Reply:
These components _are_ functional! 😄

Reply:
Looool sry haha :D should have tried it first out haha

Reply:
Fixed